enhancements/installer/openstack-supported-versions.md

```diff
@@ -0,0 +1,226 @@
+  of OpenStack and the operation is partially finished; however
+  they need to proceed to an OpenShift operation (e.g. fresh install, scale up).
+  It it possible to run an OpenStack cloud with various versions of
+  APIs, as long as compatibility constraints are respected (out of
+  scope for this document).
+- the administrators have a Support Exception and an OpenStack upgrade
+  isn't possible at the moment of the operation.
+
+In that case, we would use a new configuration flag named `ignoreVersionCheck`.
+
+### User Stories
+
+#### Check the OpenStack version
+
+As an enterprise OpenShift cluster administrator, when I run an operation
+against my OpenShift cluster, I want to make sure that the version of
+OpenStack that is running, is actually supported.
+If not, I would like to see the operation to fail early.
+
+#### Ignore the version check
+
+As an enterprise OpenShift cluster administrator, I know what I'm doing
+and need to ignore the version check in a case where I absolutely need
+to install or upgrade OpenShift.
+Performing a scale operation of an unsupported OpenStack version would
+require one of:
+
+1. The underlying OpenStack has been downgraded
+2. OCP has removed support for the underlying OpenStack
+3. The underlying OpenStack has been upgraded to an unsupported version
+
+We don't support (1), (2) isn't likely to happen and (3) would imply
+an OpenStack regression.
+Everything else being correct this should just be a sanity check so it
+shouldn't hurt, but it does add a potential source of failure.
+Therefore, ignoring the version check should only be used for a fresh
+install of OCP.
+
+##### Example Usage
+
+If the administrator wants to ignore the version validation, they would
+need to explicitly configure it, see the following example:
+
+```yaml
+apiVersion: v1
+baseDomain: example.com
+metadata:
+  name: test-cluster
+platform:
+  openstack:
+    cloud: mycloud
+    ignoreVersionCheck: true
+    ...
+pullSecret: '{"auths": ...}'
+sshKey: ssh-ed25519 AAAA...
+```
+
+### Test Plan   
+
+Unit tests and validations for this feature will be added to the installer
+to make sure that correct usage is enforced and that this feature does not
+hinder the usage of other features. To ensure GA readiness, it will be
+vetted by the QE team as well to make sure that it works with the following
+use cases:
+- running the installer against a supported version, deployment should work.
+- running the installer against an unsupported version, deployment should
+  not work.
+- running the installer against an unsupported version and set
+  `ignoreVersionCheck` to true, deployment should work.
+- upgrading OpenShift against a supported version, upgrade should work.
+- upgrading OpenShift against an unsupported version, upgrade
+  should be blocked.
+- upgrading OpenShift against an unsupported version, upgrade
+  should be not blocked if `ignoreVersionCheck` is set to true.
```

Comment:
No, this should be treated the same as the above usecase. 
`ignoreVersionCheck` should only apply at install time (and should simply skip the checks for compatible versions).